src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml

```diff
@@ -108,10 +108,10 @@
                 <MouseBinding Command="hx:ViewportCommands.Reset" Gesture="Ctrl+MiddleDoubleClick"/>
                 <MouseBinding Command="{Binding LeftClickCommand}" Gesture="LeftClick"/>
             </hx:Viewport3DX.InputBindings>
-            <hx:GroupModel3D
+            <!--<hx:GroupModel3D
```

Comment:
This binding is no longer needed as we add scene objects to the `Items` property on the Helix viewport (`Viewport3DX` object) that is already internally bound to the view.